plop-templates/components/{{kebabCase name}}/stories/{{kebabCase name}}.happo.js

```diff
@@ -0,0 +1,23 @@
+import { storiesOf } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.{{kebabCase name}}';
+
+import { createHappoStories } from '../../../../.storybook-happo/utils';
+
+
+storiesOf('ts-{{kebabCase name}}', module).add('test', () => {
+	const properties = {
+		dir: { 'rtl': 'rtl' },
+		disabled: { true: true },
+		size: sizes
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') "sizes" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS3827)